--- a/lib/additionals/patches/query_patch.rb
+++ b/lib/additionals/patches/query_patch.rb
@@ -6,17 +6,9 @@
       extend ActiveSupport::Concern
 
       included do
-        include InstanceMethods
+        prepend InstanceMethods
 
         attr_accessor :search_string
-
-        # NOTE: alias_method instead of prepend to stay compatible with redmineup redmine_contacts
-        alias_method :add_available_filter_without_additionals, :add_available_filter
-        alias_method :add_available_filter, :add_available_filter_with_additionals
-
-        # NOTE: alias_method instead of prepend to stay compatible with redmineup redmine_contacts
-        alias_method :add_filter_without_additionals, :add_filter
-        alias_method :add_filter, :add_filter_with_additionals
 
         # list_optional is default, but required for short filters
         operators_by_filter_type[:author_optional] = operators_by_filter_type[:list_optional]
@@ -45,8 +37,8 @@
       end
 
       module InstanceMethods
-        def add_filter_with_additionals(field, operator, values = nil)
-          add_filter_without_additionals field, operator, values
+        def add_filter(field, operator, values = nil)
+          super(field, operator, values)
           return unless available_filters[field]
 
           initialize_user_values_for_select2 field, values
@@ -55,8 +47,8 @@
           true
         end
 
-        def add_available_filter_with_additionals(field, options)
-          add_available_filter_without_additionals field, options
+        def add_available_filter(field, options)
+          super(field, options)
           values = filters&.dig(field, :values) || []
           initialize_user_values_for_select2 field, values
 
